whispeer/app
# verify a user.

- working barcode scanner plugin

Todo:
- add button to verify
- open barcode scanner
- check code === user fingerprint
- verify user
